src/pymorize/validate.py

```diff
@@ -72,6 +72,16 @@ def _validate(self, document):
                     "schema": {"type": "string"},
                     "required": True,
                 },
+                "enabled": {"type": "boolean", "required": False},
+                "description": {"type": "string", "required": False},
+                "pipelines": {
+                    "type": "list",
+                    # FIXME(PG): Should cross-check with pipelines.
+                    "schema": {"type": "string"},
+                },
+                "cmor_units": {"type": "string", "required": False},
+                # FIXME(PS): How is it currently defined?
+                "model_units": {"type": "string", "required": False},
```

Comment:
@siligam: This still needs to be added correctly for `units.py`. I wasn't sure how you had designed it.

Reply:
@pgierz "model_units" is referred as "source_units" in units.py. It is a optional parameter. If it is not provided, it reads the model units from xarray dataset. At times, user prefers to use some other unit instead of units from xarray dataset. In this case, "source_units" takes precedence if provided.
The setting "model_units" : {"required": False} is apt.